plugin/src/test/resources/expected_output/basic/akka/actor/dsl/Inbox.java

```diff
@@ -0,0 +1,7 @@
+package akka.actor.dsl;
+public  interface Inbox {
```

Comment:
I thought interfaces became public by default in java, but I don't mind if tooling is happy

Reply:
They do - I could also have set the access to `""` for traits/interfaces, but then also some of the existing test cases would have to be updated, so I avoided that.

Reply:
I believe Java interfaces are package-private by default, not public.

Reply:
(Perhaps you're thinking of the thing where _methods_ in interfaces don't need an explicit `public`.)

Reply:
Ah indeed, I think I was confused by the fact that traits (if I understand correctly) are always compiled into public interfaces, regardless of their own access modifiers. Added #162 to make the logic reflect that better (though the behavior remains largely the same)